README.md

```diff
@@ -74,4 +77,12 @@ we refer to collectively as "extensions") in the
 [Kong Hub](https://docs.konghq.com/hub), with documentation hosted
 on the Kong website for ready access.
 
-See [CONTRIBUTING](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#contributing-to-kong-documentation-and-the-kong-hub) for more information.
```

Comment:
This needs to not be deleted - it is specifically about contributing to the Kong Hub.

Reply:
Resolved!